codegen/__init__.py

```diff
@@ -21,14 +21,15 @@
     build_from_imports_py,
 )
 from codegen.validators import (
-    write_validator_py,
-    write_data_validator_py,
+    get_data_validator_params,
+    get_validator_params,
+    write_validator_json,
     get_data_validator_instance,
 )
 
 # Target Python version for code formatting with Black.
 # Must be one of the values listed in pyproject.toml.
-BLACK_TARGET_VERSIONS = "py38 py39 py310 py311 py312"
+BLACK_TARGET_VERSIONS = "py39 py310 py311 py312"
```

Comment:
revert this line -- needs to remain the same as what's in `pyproject.toml`
```suggestion
BLACK_TARGET_VERSIONS = "py38 py39 py310 py311 py312"
```